src/App.jsx

```diff
@@ -1,13 +1,21 @@
 import { useState } from 'react'
 import './App.css'
+import {Routes, Route} from 'react-router'
 import Header from './Components/Header'
 import NavBar from './Components/NavBar'
+import ArticleList from './Components/ArticleList'
+import ArticlePage from './Components/ArticlePage'
 
 function App() {
+  const [articles, setArticles] = useState([]);
```

Comment:
Does this state for the `articles` need to be declared in _App.jsx_ and passed down, could it not be declared in `ArticleList`?